server/meta.jsx

```diff
@@ -15,6 +15,9 @@ type SDKMeta = {|
     getSDKLoader : (options? : {| baseURL? : string, nonce? : string |}) => string
 |};
 
+const emailRegex = /^.+@.+$/;
+const emailMaxLengthRegex = /^.{1,64}$/
```

Comment:
I thought more about this and would prefer a more relaxed length restriction. 
```suggestion
const emailMaxLengthRegex = /^.{1,256}$/
```